contracts/taskModules/TriggerModule.sol

```diff
@@ -36,4 +36,29 @@ contract TriggerModule is TaskModuleBase {
 
         return abi.encode(LibDataTypes.TriggerType.CRON, triggerConfig);
     }
+
+    /**
+     * @notice Helper function to encode arguments for TriggerModule for Event.
+     *
+     * @param _address Address to listen to for events.
+     * @param _topics Array of topics to filter .
+     * @param _topicPositions Position of the topics in the array (starts from 0).
+     *
+     * e.g. - [[A,B],null,[C,D]]
+     * _topics: [A,B,C,D]
+     * _topicPositions: [0,0,2,2]
+     */
+    function encodeEventTriggerModuleArg(
+        address _address,
+        bytes32[] memory _topics,
+        uint256[] memory _topicPositions
```

Comment:
Maybe this is a very uncommon use-case and supporting this would be inefficient but, currently we cannot support the following topic condition

```[[A,B],[A,C]]```

a condition that will be triggered when A event occurs or both B and C

Reply:
User can define it like so
_topics: [A,B,A,C]
_topicPositions: [0,0,1,1]

Reply:
I thought of this case which cannot be defined with our current implementation
`[A,[B]]`  - (A AND B)
A hack is to define it like so instead `[A,[B,null]]` - (A AND (B or null))

Reply:
An alternative implementation

```
    struct EventTriggerTopic {
        bytes32[] _topics;
        bool _isArray;
    }
    
    function encodeEventTriggerModuleArg(
        address _address,
        EventTriggerTopic[4] memory _topics
    ) external pure returns (bytes memory) {
        bytes memory triggerConfig = abi.encode(
            _address,
            _topics
        );

        return abi.encode(LibDataTypes.TriggerType.EVENT, triggerConfig);
    }
```